lib/utils/util.js

```diff
@@ -0,0 +1,59 @@
+/** @flow */
+
+import {spawn} from 'child_process';
+import {window, workspace} from 'vscode'
+
+const NODE_NOT_FOUND = '[Flow] Cannot find node in PATH. The simpliest way to resolve it is install node globally'
+const FLOW_NOT_FOUND = '[Flow] Cannot find flow in PATH. Try to install it by npm install flow-bin -g'
+
+function getPathToFlow(): string {
+	return workspace.getConfiguration('flow').get('pathToFlow')
+}
+
+export function isFlowEnabled() {
+	return workspace.getConfiguration('flow').get('enabled')
+}
+
+export function checkNode() {
```

Comment:
You probably installed Flow from the `flow-bin` npm package, right? If you download the binary directly you don't need node.